api_docs/changelog.md

```diff
@@ -20,6 +20,11 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 10.0
 
+**Feature level 281**
+
+* `POST users/me/tutorial_status`: Deprecated the endpoint as it
+  is no longer in use.
```

Comment:
A quick note on terminology: "Deprecated" means that a feature still exist, but its use is no longer recommended because it will be removed in the future. "Removed" is the word you wanted here. I'll change before merging.